jsapp/js/account/accountSettingsRoute.tsx

```diff
@@ -60,8 +66,29 @@ const AccountSettings = () => {
       instagram: currentLoggedAccount.extra_details.instagram,
       newsletter_subscription:
         currentLoggedAccount.extra_details.newsletter_subscription,
-    });
-  }, [currentLoggedAccount]);
+    };
+
+    setFormFields(fields);
+
+    const fieldKeys = Object.keys(fields) as Array<keyof AccountFieldsValues>;
+
+    const organization = orgQuery.data;
+
+    // We will not display organization fields if it's and MMO organization
```

Comment:
Typo: should be "an MMO organization".

It might be more accurate to say "We will not display organization fields if user is a member of an MMO ..."